server/block/banner.go

```diff
@@ -113,6 +113,34 @@ func (b Banner) DecodeNBT(data map[string]any) any {
 	return b
 }
```

Comment:
Can EncodeBlockNBT and DecodeBlockNBT re-use EncodeItemNBT and DecodeItemNBT?

Reply:
Here, or in general? Here, probably, but not in certain blocks like shucker boxes.

Reply:
Yes, I mean here.